x-pack/plugins/enterprise_search/public/applications/workplace_search/assets/supports_acl.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" height="38" viewBox="0 0 40 38" width="40" xmlns="http://www.w3.org/2000/svg"><g fill="#98a2b3"><path d="m22.8644.135712c-.3582.285768-.2974.503496.6893 2.204498.4867.83689.9463 1.57173 1.0138 1.62616.1014.07484 1.149.09525 5.8598.09525h5.7313l.1757-.1701c.1689-.1769.1757-.20412.1757-1.32678 0-1.10225-.0068-1.15668-.1622-1.31317-.0116-.01169-.0215-.02252-.0308-.03263-.0106-.01158-.0206-.02251-.031-.03239-.1174-.11111-.3237-.12819-2.8325-.335865l-.4174-.034572c-.4024-.034327-.8644-.073271-1.3351-.112951-.9287-.078288-1.8916-.159461-2.4971-.213642-.4055-.033018-.928-.076631-1.4656-.12149-.6808-.056822-1.3857-.11565-1.9069-.157474-.4696-.040814-.9561-.081629-1.3616-.115644-.4055-.0340201-.7302-.0612599-.8755-.074868-.4799-.040824-.5542-.02721599-.7299.115668z"/><path d="m23.2429 6.04839c-1.3247.694-1.6086.87091-1.6627 1.0342-.054.14969-.0405.22454.0609.3402.1216.12928.2162.14289 1.2773.14289h1.149l.0406.73483c.1554 2.81009 2.0275 5.13699 4.7107 5.87189.5948.1633 1.8857.2177 2.548.1156 3.163-.5171 5.5083-3.3884 5.3326-6.54542-.0406-.77565-.2298-1.61255-.4867-2.15006l-.1622-.34701-5.6096-.01361-5.6029-.02041z"/><path d="m21.9587 16.962c.8178-.5443 1.8857-.973 2.8454-1.1294.0946-.0205 2.4804-.0341 5.3055-.0409 4.3863-.0068 5.123.0068 5.0757.0885-.0338.0544-2.6291 3.368-5.7718 7.3551l-5.7111 7.2599-.0338-3.6197-.0405-3.613-5.8665 8.2397-3.3252-.0068h-3.3252l1.9329-2.7284c.5901-.8363 1.5021-2.1249 2.4535-3.4692.7594-1.073 1.544-2.1816 2.21-3.1239 2.8791-4.0756 3.2779-4.5655 4.2511-5.2119z"/><path d="m37.3143 16.8736c-.196.2449-1.345 1.701-2.548 3.2319-.7493.9535-1.6113 2.0495-2.3036 2.9297l-1.0081 1.2819c-3.9403 5.0078-6.0828 7.7498-6.0828 7.7838 0 .0204 2.0479.0477 4.5553.0613l4.5621.0136v2.2861l-10.8206.0408v1.0887c0 1.1703.0609 1.4492.3718 1.8643.1014.1293.3311.3062.5069.3946.0171.0079.0333.0154.0498.0225.2941.1272.6432.1272 7.2495.1272h6.9749l.3244-.2041c.4122-.2654.6015-.5171.7367-.9594.1554-.5239.1554-16.3568.0067-17.2345-.1757-.9934-.5272-1.7282-1.1422-2.3814-.3852-.4151-.8583-.7961-.9935-.7961-.0473 0-.2433.2041-.4393.4491z"/><path d="m.196 19.1325c.38524-.7961.96648-1.2996 1.81131-1.5717.31765-.1021.64206-.1293 1.56124-.1293 1.2233-.0068 1.48689.0476 1.64234.347.04731.0884.07434 2.5855.07434 7.498v7.3619h7.27227c5.0757 0 7.3196.0273 7.4412.0817.3447.1565.4055.381.3988 1.6125 0 1.5446-.1284 2.0276-.7232 2.708-.3447.3879-.6488.5988-1.2098.8029-.4055.1565-.4393.1565-6.3531.1565-5.92051 0-5.94079 0-6.3801-.1565-.80427-.279-1.50041-.9662-1.764-1.7282-.07434-.2245-.13517-.6668-.15544-1.1635l-.0338-.7893-.82455-.0408c-.588-.0272-.93269-.0748-1.19627-.1769-.66235-.2585-1.257103-.8301-1.574758-1.5173l-.182482-.3946v-12.499z"/></g></svg>
```

Comment:
I don't think this image is used anywhere now. Could you check?

Reply:
It's used [here](https://github.com/scottybollinger/kibana/blob/scottybollinger/source-components/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/overview.tsx#L53), but I realize now that we have images in 2 locations. Will fix that now.